releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,13 @@ Upgrading Firely Server
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Firely Server.
 
+.. _announcement_vonk_08_07_2021:
+
+Public Endpoint Announcement 08.07.2021
```

Comment:
To avoid confusion on the day / month I'd use '8 July 2021'.

Reply:
Done!